client/src/board.rs

```diff
@@ -125,14 +125,31 @@ pub(crate) fn to_point(position: &Position, board_size: u32, perspective: Color)
     }
 }
 
-fn handle_on_key_down(cx: Scope<BoardProps>, event: Event<KeyboardData>, arrows: &UseRef<Arrows>) {
+fn handle_on_key_down(
+    cx: Scope<BoardProps>,
+    event: Event<KeyboardData>,
+    arrows: &UseRef<Arrows>,
+    last_move: &UseState<Option<Move>>,
+) {
     let game = use_shared_state::<Game>(cx).unwrap();
 
     match event.key() {
-        Key::ArrowLeft => game.write().go_back_a_move(),
-        Key::ArrowRight => game.write().go_forward_a_move(),
-        Key::ArrowUp => game.write().resume(),
-        Key::ArrowDown => game.write().go_to_start(),
+        Key::ArrowLeft => {
+            game.write().go_back_a_move();
+            last_move.set(game.read().get_current_move());
```

Comment:
Can we just call `get_current_move` whenever we need `last_move` and just not have the `last_move` hook at all?